applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -509,6 +517,11 @@ public void setExtraParameter(final String key, @Nullable final String value)
         }
     }
 
+    public void setInitializationAdUnitIds(final List<String> rawAdUnitIds)
+    {
+        initializationAdUnitIdsToSet = new ArrayList<>( rawAdUnitIds );
```

Comment:
Is there a reason we are creating a copy of the IDs, instead of setting them like we do on iOS?

Reply:
Since the setting is not a deep copy, I don't know what happens to the reference returned by the system.  But, the iOS copy is strong and will be kept until released.